01.py

```diff
@@ -0,0 +1,21 @@
+# 최빈수 구하기
+
+T = int(input())
+
+for t in range(1,T+1):
+    a = int(input()) 
+    score = list(map(int,input().split())) 
+    
+    num_dic = {} # 빈 딕셔너리 생성 
+
+    for i in score:
+        if i in num_dic:
+            num_dic[i] += 1  # 값 있으면 +1 
+        else:
+            num_dic[i] = 1 # 값 없으면 그냥 1 
+
+# print(dic)
+        # 이후에 딕셔너리에서 Key 값이 가장 높은 value를 꺼내오고 싶었는데 
+        # 어떻게 해야할지 모르겠어요 ㅜ
+
+    # print(f'#{a} {(max(num_dic, }')
```

Comment:
거의 다 풀어주셨네요! 👍 
딕셔너리에는 `items` 와 `values`라는 함수가 있습니다. 이 둘을 사용하시면 해결하실 수 있을 것 같아요!
```suggestion
    max_cnt = max(num_dic.values()) # 가장 많이 나온 값이 몇번 나왔는지 찾아줍니다.
    result = []
    for key, value in num_dic.items():  # num_dic[key] = value 형태
        if value == max_cnt:                   # 만일 value 값이 최대 횟수와 같다면?
            result.append(key)                 # 답이 될 수 있으므로 추가해줍시다.
    print(f'#{a} {max(result)}')             # 답이 될 수 있는 수 중 가장 큰 값을 출력해줍시다.
```
위 코드는 해결할 수 있는 방법 중 한 가지입니다! 😄